.github/workflows/ci.yml

```diff
@@ -292,3 +292,36 @@ jobs:
                  -F "force=true" \
                  "$url/api/v4/plugins"
           done
+
+  docker-mcp-server-build-push:
+    runs-on: ubuntu-latest
+    if: startsWith(github.ref, 'refs/tags/v')
+    steps:
+      - uses: actions/checkout@v4
```

Comment:
I'd consider using `commit SHA` pointing to a specific commit for better reproducibility. For example, instead of  `actions/checkout@v6.0.1` we can use something like `actions/checkout@ 8e8c483db84b4bee98b60c0593521ed34d9990e8  #v6.0.1`. The same applies for there steps.